adapters/hibernate/src/main/java/org/atomhopper/hibernate/HibernateFeedRepository.java

```diff
@@ -210,20 +217,36 @@ public void perform(Session liveSession) {
 
     @Override
     public void saveEntry(final PersistedEntry entry) {
```

Comment:
The issue also affects `updateCategories`, so a similar retry mechanism is needed there.